### PR TITLE
Rename auth0 actions

### PR DESCRIPTION
**Pull request by kmaid**
rewrite connect user action and tests

### CODE REVIEW

File: apps/auth0-rules/src/__tests__/post-login-connect-user.test.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    AUTH0_ADDITIONAL_CLAIM_DOMAIN: '',
+  },
+} as DeepPartial<Auth0PostLoginEventWithSecrets> as Auth0PostLoginEventWithSecrets;
+
+const apiBase = {
+  access: {
+    deny: jest.fn(),
+  },
+} as Partial<Auth0PostLoginApi> as Auth0PostLoginApi;
+
```
Comment (peterstarling) [inline apps/auth0-rules/src/__tests__/post-login-connect-user.test.ts:59]:
```suggestion
describe('Auth0 Action - Connect User', () => {
```

Comment (peterstarling) [reply apps/auth0-rules/src/__tests__/post-login-connect-user.test.ts:59]: I'd add a root level describe cause otherwise the output looks terrible

**Review by peterstarling — APPROVED** [review]

File: apps/auth0-rules/src/__tests__/post-login-connect-user.test.ts (hunk 1 above)
Comment (kmaid) [reply apps/auth0-rules/src/__tests__/post-login-connect-user.test.ts:59]:
``` FAIL   test-auth0-rules  apps/auth0-rules/src/__tests__/post-login-connect-user.test.ts
  ● Console

    console.log


      at onExecutePostLogin (src/post-login-connect-user.ts:27:15)

  ● should deny access  an error if fails to connect the user```

Comment (kmaid) [reply apps/auth0-rules/src/__tests__/post-login-connect-user.test.ts:59]: It has the test filename which just mean it be duplicated information but inconsistent.

**Pull request merged**